paper.md

```diff
@@ -0,0 +1,178 @@
+    orcid: 0000-0002-3535-9406
+    affiliation: "1, 3"
+  - name: Dafne van Kuppevelt
+    affiliation: 1
+  - name: Peter Steinbach
+    orcid: 0000-0002-4974-230X
+    affiliation: 4
+  - name: Berend Weel
+    orcid: 0000-0002-9693-9332
+    affiliation: 1
+  - name: Colin Sauze
+    orcid: 0000-0001-5368-9217
+    affiliation: 5
+  - name: Samantha Wittke
+    orcid: 0000-0002-9625-7235
+    affiliation: "6,7"
+  - name: Djura Smits
+    orcid: 0000-0003-4096-0260
+    affiliation: 1
+  - name: Cunliang Geng
+    orcid: 0000-0002-1409-8358
+    affiliation: 1
+  - name: Pranav Chandramouli
+    orcid: 0000-0002-7896-2969
+    affiliation: 1
+
+affiliations:
+ - name: Netherlands eScience Center, Amsterdam, The Netherlands
+   index: 1
+ - name: Simula Research Laboratory, Oslo, Norway
+   index: 2
+ - name: Düsseldorf University of Applied Sciences, Düsseldorf, Germany
+   index: 3
+ - name: Helmholtz-Zentrum, Dresden-Rossendorf, Germany
+   index: 4
+ - name: National Oceanography Centre, Liverpool, Great-Britain
+   index: 5
+ - name: CSC - IT center for Science, Espoo, Finland
+   index: 6
+ - name: Aalto University, Espoo, Finland
+   index: 7
+date: 8 August 2023
+bibliography: paper.bib
+
+---
+
+# Summary
+This article describes a hands-on introduction to the first steps in deep learning, 
+intended for researchers who are familiar with (non-deep) machine learning.
+
+The use of deep learning has seen a sharp increase of popularity and applicability over the last decade. 
+While deep learning can be a useful tool for researchers from a wide range of domains, 
+taking the first steps in the world of deep learning can be somewhat intimidating. 
+This introduction aims to cover the basics of deep learning in a practical and hands-on manner, 
+so that upon completion, students will be able to train their first neural network and understand 
+what next steps to take to improve the model.
+
+The lesson starts with explaining the basic concepts of neural networks, 
+and then go through the different steps of a deep learning workflow. 
+Learners will learn how to prepare data for deep learning, 
+how to implement a basic deep learning model in Python with Keras, 
+how to monitor and troubleshoot the training process and how to implement different layer types such as convolutional layers.
+
+# Statement of Need
+There are many great free online course materials on deep learning, 
+for example: fast.ai's Practical Deep Learning for Coders, MORE ONLINE COURSES
+
+Nonetheless, these resources are often not available open-source and can thus not be easily adapted to the students' needs. 
+Also, the amount and diversity of the existing online courses is overwhelming, but they are generally not targeted towards academic researchers.
+What works well for researchers is to both make them familiar with the key concepts, and also let them 
+practice with how to implement it. Eventually resulting in a feeling of 'I can do this myself'. 
```

Comment:
```suggestion
practice with how to implement it. Eventually resulting in an increase in confidence and  the conviction that 'I can do this myself'. 
```

Reply:
I am not convinced that this statement is true only of researchers. It seems like the same description could apply to anyone who wants to learn enough DL to be able to use it in their work.